### ISSUE TITLE
Show transaction history better

### ISSUE THREAD

**Issue by jellegerbrandy** [Paratii-Video/paratii-player#138]
The Transactions colleciton in meteor adds a record for each Logged event.

I.e. a "buyVideo" is a single transaction that logs 3 separate events:

- BuyVideo - logs the videoId and price of the video
- Transfer - logs the transfer of PTI from the buyers account to the owner of the video
- Transfer - logs the transfer of PTI from the buyers account to the distribution pool

At the moment, we are showing: each line in the history in the app corresponds to 1 transaction, and we are just piling up the info from the different events that are logged for each transaction

We (probably) just want to show the info of the first event, in this case.

**Comment by pedrocasa** [comment]
What if we show the "main action" ( buying a video for example) and when the user clicks on that information we show the other 3 events? Wouldn't it be more simple than showing all the little transactions that a main action has in another page ( after clicking on the main action ) ?

ps: I'm sorry, I couldn't find what you meant when you said "meteor collection"

**Comment by jellegerbrandy** [comment]
Yes, we can do that. But, after giving it some thought, we can just show a simple description of the "main event" (in this case the buyvideo thing) and ignore the others info for now to avoid the complexity.

("meteor collection" -> should have said "mongodb collection". It is the database table where the application gets the data for the transaction history from.)
